examples/grpc/proto-util.arrai

```diff
@@ -10,11 +10,20 @@ let grpc = //./grpc;
     endpointInfo: \ep
         let method = //.str.sub(//.str.title(//.str.lower(ep.name)), "-", "");
         let paramName = method + "Request" if wrap else $"${ep.params >> grpc.type(.)::, }";
-        let responseName = method + "Response" if wrap else ep.ret("ok");
+        let streamRes = //.str.sub(ep.ret("ok"), "sequence of", "stream") if ep.attrs.stream else //.str.sub(ep.ret("ok"), "sequence of ", "") + "s";
```

Comment:
Would this suffice?
```suggestion
        let streamRes = //.str.sub(ep.ret("ok"), "sequence of ", "stream" if ep.attrs.stream) + "s";
```

Reply:
Not really this appends "s" for both if and else, but I want it only for else.